docs/src/content/docs/get-started/url-shortner.mdx

```diff
@@ -0,0 +1,306 @@
+
+1. *Start DiceDB*: Ensure DiceDB is running.
+2. *Test the API*:
+    - Shorten URL:
+        Send a POST request to `/shorten` with JSON body on Postman:
+        ```
+        {
+            "long_url": "https://example.com"
+        }
+        ```
+
+        OR
+
+        ```curl
+        curl -X POST -H "Content-Type: application/json" 
+            -d '{"long_url": "https://example.com"}' http://localhost:8080/shorten
+        ```
+
+    - Redirect to Original URL:
+        Send a GET request to `/:id` with the short URL ID on Postman
+
+        OR
+
+        ```curl
+        curl -L http://localhost:8080/{short_id}
+        ```
+
+## DiceDB Commands Used
+
+Here are the main DiceDB commands we’ll use to store and retrieve URLs.
+
+1. `Set` Command: Stores a key-value pair in DiceDB.
+Syntax - `Set(key, value, expiration)`
+`key`: Unique identifier (e.g., short URL code)
+`value`: The data to store (serialized JSON)
+`expiration`: Optional; 0 means no expiration
+
+2. `Get` Command: Retrieves the value associated with a key.
+Syntax - `Get(key)`
+`key`: The identifier for the data to retrieve.
+
+## Project Structure
+
+The project is organized into several packages:
+
+- `models`: Defines the URL data structure.
+- `repository`: Handles database interactions with DiceDB.
+- `services`: Implements business logic for URL shortening.
+- `handlers`: Contains HTTP route handlers for the API.
+- `utils`: Provides helper functions for API responses.
+
+## Code overview
+
+1. `models/url.go`: Defines the URL struct, which represents the data to be stored in DiceDB.
+
+    ```go
+    package models
+
+    type URL struct {
+        ID       string `json:"id"`
+        LongURL  string `json:"long_url"`
+        ShortURL string `json:"short_url"`
+    }
+    ```
+
+    - This struct will be serialized to JSON before saving in DiceDB.
+
+2. `repository/url_repository.go`: Handles interactions with DiceDB, using `Set` and `Get` commands to store and retrieve URL data.
+
+    ```go
+    package repository
+
+    import (
+        "context"
+        "encoding/json"
+        "errors"
+        "fmt"
+        "url-shortner-dicedb/models"
+
+        "github.com/dicedb/dicedb-go" // DiceDB Go SDK
+    )
+
+    var db *dicedb.Client
+
+    func init() {
+        db = dicedb.NewClient(&dicedb.Options{
+            Addr: "localhost:7379", 
+        })
+    }
+
+    // SaveURL stores a URL in DiceDB
+    func SaveURL(url models.URL) error {
+        ctx := context.Background()
+
+        // Serialize the URL struct to JSON
+        urlData, err := json.Marshal(url)
+        if err != nil {
+            return fmt.Errorf("failed to serialize URL: %w", err)
+        }
+
+        // Store the serialized URL JSON in DiceDB
+        statusCmd := db.Set(ctx, url.ID, urlData, 0)
+        _, err = statusCmd.Result()
+        if err != nil {
+            return fmt.Errorf("failed to save URL: %w", err)
+        }
+        return nil
+    }
```

Comment:
Based on above comments this can be simplified down to just a couple of lines of code. All it needs to do is store short URLs mapped to long URLs.